src/aoc2020/day1.clj

```diff
@@ -0,0 +1,143 @@
+(ns aoc2020.day1
+  (:require [aoc2020.util :refer [file->seq]]))
+
+
+;; part 1
+
+(defn strs->ints [strs]
+  "parsing strings to integers"
+  (let [str->int (fn [str]
+                   (Integer/parseInt str))]
+    (map str->int strs)))
+
+(defn int->compl [n sum]
+  "calculate complement"
+  (- sum n))
+
+(defn compl-in-set? [n sum s]
```

Comment:
compl is a domain idea.
set is a weak type

Reply:
The ordering of these args is off

Also, this function likely doesn't need to exist

Reply:
> The ordering of these args is off
> 
> Also, this function likely doesn't need to exist

"The ordering of these args is off" means the order of these 3 args is wrong?

Reply:
> compl is a domain idea. set is a weak type

how about the name `compl-in-entries`?

Reply:
> > compl is a domain idea. set is a weak type
> 
> how about the name `compl-in-entries?` or `compl-exists?`

`compl-in-entries?` would reflect the domain and is a much stronger name.